doc/book/zend.ldap.introduction.md

```diff
@@ -0,0 +1,195 @@
+# Introduction to Zend\\Ldap
+
+`Zend\Ldap\Ldap` is a class for performing *LDAP* operations including but not limited to binding,
+searching and modifying entries in an *LDAP* directory.
+
+## Theory of operation
+
+This component currently consists of the main `Zend\Ldap\Ldap` class, that conceptually represents a
+binding to a single *LDAP* server and allows for executing operations against a *LDAP* server such
+as OpenLDAP or ActiveDirectory (AD) servers. The parameters for binding may be provided explicitly
+or in the form of an options array. `Zend\Ldap\Node` provides an object-oriented interface for
+single *LDAP* nodes and can be used to form a basis for an active-record-like interface for a
+*LDAP*-based domain model.
+
+The component provides several helper classes to perform operations on *LDAP* entries
+(`Zend\Ldap\Attribute`) such as setting and retrieving attributes (date values, passwords, boolean
+values, ...), to create and modify *LDAP* filter strings (`Zend\Ldap\Filter`) and to manipulate
+*LDAP* distinguished names (DN) (`Zend\Ldap\Dn`).
+
+Additionally the component abstracts *LDAP* schema browsing for OpenLDAP and ActiveDirectory servers
+`Zend\Ldap\Node\Schema` and server information retrieval for OpenLDAP-, ActiveDirectory- and Novell
+eDirectory servers (`Zend\Ldap\Node\RootDse`).
+
+Using the `Zend\Ldap\Ldap` class depends on the type of *LDAP* server and is best summarized with
+some simple examples.
```

Comment:
Note to @heiglandreas: Rephrase that to make clear that all LDAP-Servers are handled equally and that there might be special options to make working with certain Directory-Servers easier (like the ```bindRequiresDn``` for AD) - Check also the default value for that option